starboard/android/apk/app/src/main/java/dev/cobalt/libraries/services/clientloginfo/ClientLogInfo.java

```diff
@@ -32,7 +33,11 @@ public void afterStopped() {}
   public ResponseToClient receiveFromClient(byte[] data) {
     ResponseToClient response = new ResponseToClient();
     response.invalidState = false;
-    response.data = clientInfo.getBytes(UTF_8);
+
+    String responseString =
+        "displayRefreshRate:" + VideoFrameReleaseTimeHelper.getDefaultDisplayRefreshRate() + ";";
```

Comment:
Will the client call this multiple times? Should displayRefreshRate be cached?

Reply:
It will be called per playback to get the latest refresh rate. As it's quite simple and no performance concern, I don't think it needs to be cached.